OpenTelemetry.sln

```diff
@@ -141,7 +135,13 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "OpenTelemetry.Instrumentati
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "TestApp.AspNetCore.2.1", "test\TestApp.AspNetCore.2.1\TestApp.AspNetCore.2.1.csproj", "{7CDA3F85-EA6F-45C4-B432-2559303355D4}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "OpenTelemetry.Exporter.ZPages.Tests", "test\OpenTelemetry.Exporter.ZPages.Tests\OpenTelemetry.Exporter.ZPages.Tests.csproj", "{98F9556B-116F-49B5-9211-BB1D418446FF}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "OpenTelemetry.Exporter.ZPages.Tests", "test\OpenTelemetry.Exporter.ZPages.Tests\OpenTelemetry.Exporter.ZPages.Tests.csproj", "{98F9556B-116F-49B5-9211-BB1D418446FF}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "OpenTelemetry.Exporter.AspNet.Win", "samples\AspNet\OpenTelemetry.Exporter.AspNet.Win.csproj", "{9A4E3A68-904B-4835-A3C8-F664B73098DB}"
```

Comment:
The project name looks weird to me. I guess it should be something like `OpenTelemetry.Example.Something` or `OpenTelemetry.Sample.Something`? (or just `Something` without the `OpenTelemetry` prefix).

Reply:
OpenTelemetry.Sample.AspNetCore, OpenTelemetry.Sample.AspNet, OpenTelemetry.Sample.Console - how about these/

Reply:
That sounds good to me.
The only _minor_ concern I have is that these samples share the `OpenTelemetry` namespace, so they might get access to OpenTelemetry namespace without explicit `using` statement - e.g. a customer might ask "I've copied the code from your example project and got a compilation failure" and we replied "you should add `using OpenTelemetry`".

Reply:
pushed the new renaming. let me know what do u think!